th.json

```diff
@@ -1347,31 +1348,31 @@
         "key": "D0B2C8BA4674F0C2C3F6CAB676B10CBF",
         "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(13).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
         "id": "The Spooky",
-        "str": "น่ากลัว"
+        "str": "กระตุกจิต"
     },
     {
         "key": "D0C5B7DC4D0EDAB37A13E7A562E45ED6",
         "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(3).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
         "id": "The Chilling",
-        "str": "ความเยือกเย็น"
+        "str": "เย็นชา"
     },
     {
         "key": "D2BAA1EC4F84BE5AA48BBC92F65D2FA8",
         "location": "/Game/enums/actions.actions.DisplayNameMap(9 - Value).DisplayNameMap",
         "id": "Light",
-        "str": "แสง"
+        "str": "แสงไฟ"
     },
     {
         "key": "D2EAFC9348E348C7516300BF122B9702",
         "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_5.label_17_E97135F84728B1D3E2D91DAE099E4D25",
         "id": "Right Place",
-        "str": "ที่ทางขวา"
+        "str": "พื้นที่ทางขวา"
     },
     {
         "key": "D43847EE40A03CCE1238E0A4BAB8F812",
         "location": "/Game/UI/intro.intro_C:ExecuteUbergraph_intro [Script Bytecode]",
         "id": "Translated by: {people}",
-        "str": "แปลโดย: {people}"
+        "str": "แปลโดย: {people} "
```

Comment:
is correct the white space at the end?

Reply:
Oh no there will be no space at the back, sorry!